charts/gluu/charts/config/README.md

```diff
@@ -59,6 +59,11 @@ Kubernetes: `>=v1.21.0-0`
 | configmap.cnSqlDbPort | int | `3306` | SQL database port. |
 | configmap.cnSqlDbTimezone | string | `"UTC"` | SQL database timezone. |
 | configmap.cnSqlDbUser | string | `"gluu"` | SQL database username. |
+| configmap.cnSqlSslCaCert | string | `""` | Base64-encoded string of CA certificate used to sign client/server certificate of MySQL/PostgreSQL server. Required if using client cert authentication. |
+| configmap.cnSqlSslClientCert | string | `""` | Base64-encoded string of client certificate signed by CA. Required if using client cert authentication. |
+| configmap.cnSqlSslClientKey | string | `""` | Base64-encoded string of client key signed by CA. Required if using client cert authentication. |
+| configmap.cnSqlSslEnabled | bool | `false` | Enforce connection to SQL database using SSL. |
+| configmap.cnSqlSslMode | string | `""` | Mode when connecting to SQL database using SSL. If using MySQL, choose one of `PREFERRED`, `REQUIRED`, `VERIFY_CA`, or `VERIFY_IDENTITY`. If using PostgreSQL, choose one of `allow`, `prefer`, `require`, `verify-ca`, or `verify-full`. |
```

Comment:
Fix wording and add a security note around private keys.

- “client key signed by CA” is incorrect; keys aren’t signed. Say “client private key corresponding to the client certificate.”
- Consider adding a short warning not to commit real private keys in values; ensure they’re sourced securely and only rendered into Kubernetes Secrets (never ConfigMaps or logs).


Apply this edit:

```diff
-| configmap.cnSqlSslClientKey | string | `""` | Base64-encoded string of client key signed by CA. Required if using client cert authentication. |
+| configmap.cnSqlSslClientKey | string | `""` | Base64-encoded client private key corresponding to the client certificate. Required if using client cert authentication. |
```